### DISCUSSION TITLE
该项目可以让E5自动续订吗

### DISCUSSION THREAD

**Discussion by LouisTay** [spencerwooo/onedrive-vercel-index#320] [Q&A] [answered]
我看这个项目目前都不需要像以前那样去Azure Active Directory里面生成ID、secret之类的，还能实现E5订阅的自动续订嘛

**Comment by myl7** [comment, marked as answer]
E5 的续订是由微软自己决定的。频繁的、合理程度内的 API 使用按官方说明是会增大续订概率，此项目也会调用相应的 API，是符合前面的说明、应当增大续订概率的，但是终归是没有保证的。所以不存在“实现E5订阅的自动续订”这个说法。
另：个人的一些经验和听闻，有人完全不用也能续订，有人积极开发也会被撤权。要追求稳定买订阅才是唯一方法。

**Comment by spencerwooo** [comment]
原来……你们部署这东西的最大用处是用来续订 E5 嘛

**Reply by Lynricsy** [reply]
> 原来……你们部署这东西的最大用处是用来续订 E5 嘛

大部分人应该不是，毕竟现在在Github上面有提交都可以让E5续订，没必要弄这个，反正我主要是用来分享文件

**Reply by LouisTay** [reply]
其实我主要是来玩域名邮箱的.....
